### DISCUSSION TITLE
b 站新版界面属实让我蚌埠住了

### DISCUSSION THREAD

**Discussion by FCSDFSDVXDFG** [the1812/Bilibili-Evolved#2629] [综合讨论 / General]
长辈模式，大字体超高清！UI更新，丑出新境界！

讲个笑话，我原本准备点一下新版反馈说明问题，看到“播放器使用体验变差看”把我笑死，专门截个图

![Snipaste_2021-12-09_20-57-29](https://user-images.githubusercontent.com/65455414/145400901-b0d6fc57-b00d-4aaa-8b19-e8e717a47ef3.png)

**Comment by FoundTheWOUT** [comment]
吓我一跳，看标题还以为你说 V2 的界面8行，我寻思这么优秀的设计（甚至怀疑1812是不是读美术的）怎么还有喷的，点进来发现原来是说啊B🥲

**Reply by the1812** [reply]
我把标题改了下, 不然确实挺吓人(

**Reply by lacafjh** [reply]
感谢所有的贡献者，v2 重构后的侧栏 UI 太好看了，设计风格和 Win11 也很搭，现代的感觉。

**Reply by RingingVolcano** [reply]
1812佬的UI做得确实挺强
